src/commands/unity-symbols/README.md

```diff
@@ -28,20 +28,27 @@ To override the full URL for the intake endpoint, define the `DATADOG_SOURCEMAP_
 
 ### `upload`
 
-This command uploads your IL2CPP mapping file and iOS dSYMs to Datadog in order to deobfuscate and symbolicate your application's stack traces on iOS. Note that
-Android NDK symbol files, Proguard mapping files, and IL2CPP mapping are handled by the [Datadog Android Gradle Plugin](https://github.com/DataDog/dd-sdk-android-gradle-plugin).
+This command uploads your IL2CPP mapping file, iOS dSYMs, Android .so, and Android Proguard mapping files to Datadog in order to deobfuscate and symbolicate your application's stack traces.
 
-After performing an iOS build, the Datadog Unity SDK copies all required files to your build output directory under `datadogSymbols`. You are then able to
-run the following command to upload all the necessary files:
+After performing a build, run the following command from your exported build directory to upload all the necessary files:
 
+For iOS:
 ```bash
-datadog-ci unity-symbols upload --symbols-location datadogSymbols
+datadog-ci unity-symbols upload --ios
+```
+
+For Android:
+```bash
+datadog unity-symbols upload --android
 ```
 
 | Parameter | Condition | Description |
 |-----------|-----------|-------------|
-| `--symbols-location`  | Optional  | The location of of your dSYMs, `build_id` and `LineNumberMappings.json` file.  Defaults to `datadogSymbols`. |
+| `--ios` or `--android` | One is required | Specify the platform we are uploading symbols for, iOS or Android |
+| `--variant` | Optional, Android Only | Specify the variant that was built. Defaults to `release` |
+| `--symbols-location`  | Optional, iOS Only | The location of of your dSYMs, `build_id` and `LineNumberMappings.json` file.  Defaults to `datadogSymbols`. |
```

Comment:
```suggestion
| `--variant` | Optional, Android only | Specify the variant that was built. Defaults to `release`. |
| `--symbols-location`  | Optional, iOS only | The location of your dSYMs, `build_id`, and `LineNumberMappings.json` file. Defaults to `datadogSymbols`. |
```

Reply:
👍 Though actually you've pointed out a mistake. `--symbols-locaiton` is no longer iOS only